packages/web-components/src/components/multi-select/multi-select.ts

```diff
@@ -150,6 +151,12 @@ class CDSMultiSelect extends CDSDropdown {
       .join(',');
   }
 
+  // Keep the menu open for individual selections, close only when clearing.
+
+  protected _shouldCloseAfterSelection(item?: CDSDropdownItem) {
```

Comment:
```suggestion
  protected _shouldCloseAfterSelection(item?: CDSMultiSelectItem) {
```
I think we can just use `CDSMultiSelectItem` here